### DISCUSSION TITLE
When I try to use typegraphql-prisma with nextjs serverless, I got MaxListenersExceededWarning

### DISCUSSION THREAD

**Discussion by Bakuhub** [MichalLytek/typegraphql-prisma#261] [Q&A]
I got the following errors when I try to run /api/graphql.

 (node:157944) MaxListenersExceededWarning: Possible EventEmitter memory leak detected. 11 SIGTERM listeners added to [process]. Use emitter.setMaxListeners() to increase limit

```

export async function createSchema() {
    return await buildSchema({
        resolvers: resolvers,
        //emitSchemaFile: process.env.NODE_ENV==="development",
        validate: false,
    });
}

export default cors(async function handler(
        req: MicroRequest, res: ServerResponse
) {
    if (req.method==="OPTIONS") {
        res.end();
        return false;
    }
    const apolloServer = new ApolloServer({
        schema: await createSchema(),
        context:  {
        prisma,
    },
    });
    const startServer = apolloServer.start();
    await startServer;
    return await apolloServer.createHandler({
        path: "/api/graphql",
    })(req, res);
});

```

My guess is the createSchema trigger every time I call the endpoint, is there any way to move the await function out, so the await createSchema() will only trigger once?

**Comment by MichalLytek** [comment]
you should build the schema once, assign to the root scope variable (or even a whole http handler) and just call it in the handler function (or build if not created yet)
